PendingReleaseNotes.md

```diff
@@ -1,7 +1,5 @@
 # Major Themes
-
```

Comment:
Please remove the whitespace / new line changes in this file.

Reply:
ack